kubernetes/resource_kubernetes_annotations_test.go

```diff
@@ -32,6 +32,7 @@ func TestAccKubernetesAnnotations_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "kind", "ConfigMap"),
 					resource.TestCheckResourceAttr(resourceName, "metadata.0.name", name),
 					resource.TestCheckResourceAttr(resourceName, "annotations.%", "0"),
+					resource.TestCheckResourceAttr(resourceName, "field_manager", "tftest"),
```

Comment:
I would also add a test with empty value for `field_manager`

Reply:
Because I've used a validator from the SDK above I don't think we need to add another test case for it